app/templates/_custom_meta.html

```diff
@@ -0,0 +1,3 @@
+{% block custom_meta %}
```

Comment:
I'm not sure I understand why this is in a `{% block %}` - if on some page we decide we want to add additional meta tags, we wouldn't want custom dimension taken out, would we? Maybe I'm missing the point here... also I wonder if this should be in the FE-toolkit?

Reply:
That's a good spot. (The pattern's copied from the Buyer FE where we set the custom meta on one particular page.) 

We should do something like this instead:
```
_custom_meta.html:

<meta name="ga_customDimension" data-id="10" data-value="{{ current_user.role }}">
{% block custom_meta %}{% endblock %}
```